@navikt/core/css/darkside/baseline/index.css

```diff
@@ -0,0 +1,27 @@
+@import "./fonts.css";
+
+/**
+ * Baseline
+ * Missing from "original"
+ * - CSS-resets
+ * - Print styles
+ * - utilities (sr-only, animation-reset etc)
+ * - Tokens for: font, font-size, line-height, shadows etc
```

Comment:
Won't we need those tokens? 🤔

Reply:
Yes, but can be added in separate PR(s) 🚀